auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -49,6 +49,7 @@ public enum Entitlement {
     CDP_CCM_V2,
     CDP_CCM_V2_JUMPGATE,
     CDP_CCM_V2_USE_ONE_WAY_TLS,
+    VERTICAL_SCALE,
```

Comment:
is it a good idea to handle all providers with one flag?

Reply:
CDP_CB_AWS_VERTICAL_SCALE now. thanks